geomapper/geomapper.py

```diff
@@ -99,16 +99,26 @@ def get(self, input_proto):
         self._transform_detections(sae_msg, camera)
 
         return self._pack_proto(sae_msg)
-
+      
     def _add_cam_location(self, sae_msg: SaeMessage):
         '''Add camera location data, if location is configured (independent of the passthrough setting)'''
         stream_id = sae_msg.frame.source_id
-
-        cam_lat = self._cam_configs[stream_id].pos_lat
-        cam_lon = self._cam_configs[stream_id].pos_lon
-        if cam_lat is not None and cam_lon is not None:
-            sae_msg.frame.camera_location.latitude = cam_lat
-            sae_msg.frame.camera_location.longitude = cam_lon
+        
+        if self._config.mapping_strategy.mode == 'static':
+            cam_lat = self._cam_configs[stream_id].pos_lat
+            cam_lon = self._cam_configs[stream_id].pos_lon
+            if cam_lat is not None and cam_lon is not None:
+                sae_msg.frame.camera_location.latitude = cam_lat
+                sae_msg.frame.camera_location.longitude = cam_lon
+        
+        if self._config.mapping_strategy.mode == 'dynamic':
+            if sae_msg.frame.camera_location != None or sae_msg.frame.camera_location.latitude != 0.0 :
```

Comment:
Do we really need the extra precaution of checking the latitude value? (also, we could use `sae_msg.frame.HasField('camera_location')` but that is just a technicality)